docs/source/migration-from-truffle-guide.rst

```diff
@@ -75,38 +83,78 @@ Modify tests
 
 *with Etherlime*
 ::
+    // step1: require Etherlime module
+    const etherlime = require('etherlime')
+
+    // step2: require compiled contract from ./build not the .sol file (as in deployment scripts)
+    const LimeFactory = require('../build/LimeFactory.json')
+
+
+    // step4: replace 'contract' descriptor to 'describe' then remove (accounts) param in async function 
     describe('LimeFactory tests', async () => {
+
+        // step5: initialize account
+        let owner = accounts[0];
+
+        // step6: set the deployer in before/beforeEach and fix the deployment scripts as we did before
         beforeEach(async function() {
-            deployer = new etherlime.EtherlimeGanacheDeployer(accounts[0].secretKey);
+
+            deployer = new etherlime.EtherlimeGanacheDeployer(owner.secretKey);
             limeFactory = await deployer.deploy(LimeFactory);
+
         });
 
         it('should do something', () => {
 
         })
     })
 
+Flexibility
+~~~~~~~~~~~
+
 - **in case you want to use an address of an account, you must extend it to** ``let owner = accounts[0].wallet.address``
-- **remove “.call” when calling view functions.** You may try to replace all at once with “.call(“  => “(“. Be careful!
-- **if you need to assert the result is an address, you may use** ``assert.isAddress(result)``
-- **when you expect a function to revert, instead of “try-catch”, use** ``assert.revert()``
-- **when a contract’s method is called, the default sender is set to accounts[0]. If you want to execute it from another account, replace** ``{from: owner}`` object with ``.from(anotherAccount)``. 
+- **when a contract’s method is called, the default sender is set to accounts[0]. If you want to execute it from another account, replace** ``{from: anotherAccount}`` object with ``.from(anotherAccount)``. 
 
+*with Truffle*
 ::
-    
-    await assert.revert(limeFactory.from(2).createLime('newLime' 0, 10, 12));
+    await limeFactory.createLime(newLime' 0, 10, 12, {from: accounts[1]})
+
 
+*with Etherlime*
+::
+    await limeFactory.from(2).createLime('newLime' 0, 10, 12);
 
-As a param you may also use: 
-    - .from(2)
-    - .from(accounts[2])
-    - .from(accounts[2].wallet)
-    - .from(accounts[2].wallet.address)
-    - .from(customAddress)
-    - .from(customWallet)
+    // as a param you may also use:
+    await limeFactory.from(accounts[1]).createLime('newLime' 0, 10, 12);
+    await limeFactory.from(accounts[1].wallet).createLime('newLime' 0, 10, 12);
+    await limeFactory.from(accounts[1].wallet.address).createLime('newLime' 0, 10, 12);
+    await limeFactory.from(customAddress).createLime('newLime' 0, 10, 12);
```

Comment:
This wont work if this is not the address of one of the default wallets as we do not have the private key of it